File: frontend/src/app/modules/menu/components/news/news.ts
export class News{
  idNews: string;
  date: string;
  title: string;
  text: string;
  img: string;


  static cloneBase(news: News): News {
    const clonedNews: News = new News();
    clonedNews.idNews = news.idNews;
    clonedNews.date = news.date;
    clonedNews.title = news.title;
    clonedNews.text = news.text;
    clonedNews.img = news.img;
    return clonedNews;
  }
}

File: fapi/src/main/java/com/gym/fapi/models/News.java
package com.gym.fapi.models;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

import java.sql.Date;

@JsonIgnoreProperties(ignoreUnknown = true)
public class News {

    private int idNews;
    private String text;
    private Date date;
    private String title;
    private String img;

    public News() {
    }

    public News(int idNews, String text, Date date, String title, String img) {
        this.idNews = idNews;
        this.text = text;
        this.date = date;
        this.title = title;
        this.img = img;
    }

    public int getIdNews() {
        return idNews;
    }

    public void setIdNews(int idNews) {
        this.idNews = idNews;
    }

    public String getText() {
        return text;
    }

    public void setText(String text) {
        this.text = text;
    }

    public Date getDate() {
        return date;
    }

    public void setDate(Date date) {
        this.date = date;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getImg() {
        return img;
    }

    public void setImg(String img) {
        this.img = img;
    }
}
